### ISSUE TITLE
jira: Support PATs(Personal Access Tokens)

### ISSUE THREAD

**Issue by qiankehan** [psss/did#258]
I tried PATs with **auth_username** and **auth_password**, but it cannot work
```
 ERROR  403 Client Error: Forbidden for url: https://XXXX/rest/api
 ERROR  Jira authentication failed. Try kinit.
```
Jira doc for PATs: https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html

**Comment by psss** [comment]
@lukaszachy has implemented support for token authentication in #259.

**Comment by AloisMahdal** [comment]
Great work! I've juist configured it locally; works like a charm!  🎉
